File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        int counter = 0;
        for(int i = 0 ; i<nums.Length; i++)
        {
            for(int j=i+1; j<nums.Length; j++)
            {
            if(nums[i]== nums[j])
            {
                counter= counter+1;
                return true;
            }
            if (j+1==nums.Length)
             break;
            }

        }
        return false;
        
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {

        HashMap<Integer, Integer> map = new HashMap<>();


        for(int i : nums)
        {
            map.put(i,map.getOrDefault(i,0)+1);

        }
        List<Map.Entry<Integer, Integer>> list = new ArrayList<>(map.entrySet());
        list.sort((a,b)-> b.getValue() - a.getValue());

        int[] answer = new int[k];

        for(int i = 0 ; i<k; i++)
        {
            answer[i]= list.get(i).getKey();
        }

        
        return answer;
    }
}
